File: dart/bigL.dart
void draw() {
  int num = 2147483647;

  int com1width = (num / 10).floor();

  for (int i = 0; i < num; i++) {
    print("L" * com1width);
  }

  int com2width = (num / 5).floor();
  int com2height = (num / 50).floor();

  for (int i = 0; i < com2height; i++) {
    print("L" * com2width);
  }

}

File: typescript/bigL.ts
export function draw(): void {
    
    const num: number = 2147483647
    
    let com1width: number = Math.floor(num / 10);
    for (let i = 0; i < num; i++) {
        console.log("L".repeat(com1width));
    }
    let com2width: number = Math.floor(num / 5);
    let com2height: number = Math.floor(num / 50);
    for (let i = 0; i < com2height; i++) {
        console.log("L".repeat(com2width))
    }
}
